app/controllers/v2/lookups_controller.rb

```diff
@@ -0,0 +1,17 @@
+module V2
+  class LookupsController < ApplicationController
+    def lookup_by_base_paths
+      if base_paths.length > 100
+        raise CommandError.new(code: 400, message: "base_paths must contain less than 100 items")
+      end
+
+      render json: Queries::LookupByBasePaths.call(base_paths)
+    end
+
+  private
+
+    def base_paths
+      params.fetch(:base_paths)
```

Comment:
maybe we could use [Strong Parameters](https://github.com/rails/strong_parameters) here (disclaimer: it's a been a while since I used them so this could be a duff suggestion) - perhaps they'd allow us to require an array of base_paths which would save us worrying about a couple of edge cases of bad usage?